x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/machine_learning.ts

```diff
@@ -335,11 +342,13 @@ export default ({ getService }: FtrProviderContext) => {
       it('should be enriched alert with criticality_level', async () => {
         const { previewId } = await previewRule({ supertest, rule });
         const previewAlerts = await getPreviewAlerts({ es, previewId });
-        expect(previewAlerts.length).toBe(1);
-        const fullAlert = previewAlerts[0]._source;
 
-        expect(fullAlert?.['host.asset.criticality']).toBe('medium_impact');
```

Comment:
was `host.asset.criticality` removed or it is not relevant in this case?

Reply:
The alert(s) in question here never had a `host.name`, so I think this was the case of a copy/paste from elsewhere that was never validated because the test had never been run. I went into detail on https://github.com/elastic/kibana/pull/181918/commits/934cb4935066bd3536f1f5336cf4dbeb577a588c.